examples/basic.js

```diff
@@ -1,24 +1,20 @@
 const { Sender } = require('@questdb/nodejs-client');
 
 async function run() {
-  // create a sender with a 4KB buffer
-  const sender = new Sender({ bufferSize: 4096 });
-
-  // connect to QuestDB
-  // host and port are required in connect options
-  await sender.connect({ port: 9009, host: 'localhost' });
+  // configure the sender
+  const sender = Sender.fromConfig('http::addr=localhost:9000');
```

Comment:
This change will break ILP/TCP snippets on the website and in Cloud UI. https://github.com/questdb/nodejs-questdb-client/blob/main/examples.manifest.yaml should be updated, as well as the website and Cloud UI code.

It'd be simpler to keep the examples as they are and introduce config-driven example, so that we can switch the website and Cloud UI to that one.

Reply:
I thought the idea was not to show examples for TCP anymore as a push to get new users to start with HTTP instead.
The removal of TCP examples (by replacing it with HTTP) was intentional.
I thought this is what we did for other clients too, but will check.
I can add it back, if this is not the case.

The yaml file definitely needs updating if the examples change, agree.

Reply:
For Go client we introduced separate sets of TCP and HTTP examples, as well as a from-conf example. The structure can be seen here: https://github.com/questdb/go-questdb-client/tree/main/examples

In other clients, it's a bit different, but that's fine as long as we keep the examples usable by the website and Cloud UI. So probably it's ok that convert all examples to conf, but we need to update the manifest file - as far as I can see Cloud UI now uses the `ilp-from-conf` snippet while for the website it's not so obvious.

Reply:
changed the examples back, and added the extra one: ilp-from-conf